scripts/run_docker_build.sh

```diff
@@ -16,6 +16,7 @@ conda-build:
 
 always_yes: true
 show_channel_urls: true
+ssl_verify: false
```

Comment:
You don't need to run that script. I have the same problem so I use the docker image interactively (which is actually better for debugging): `docker run -it --rm -v $(pwd):/home/ condaforge/linux-anvil /bin/sh`